Rakefile

```diff
@@ -76,4 +76,12 @@ task :coverity do
   FileUtils.rm_rf(['./cov-int', 'cov-fluentd.tar.gz'])
 end
 
+desc 'Build SBOM files'
+task :sbom do
+  require 'fluent/version'
+  version = ENV["SBOM_VERSION"] || Fluent::VERSION
+  sh "docker sbom fluent/fluentd:v#{version}-debian-amd64-1.0 --output licenses/fluentd-latest.spdx.json --format spdx-json"
```

Comment:
I'm just using with sbom subcommand which is included in Docker Desktop 4.24.2 (124339).

Reply:
If the generated SBOM files are not comfortable for y'all, I'll remove the commit https://github.com/fluent/fluentd/pull/4356/commits/8cb884fb74f6d25e8b147a753df6bf40b7015aa9 from this PR. What do you think?

Reply:
Basically, it is good thing to provide SBOM, but I'm not sure it should be managed in fluent/fluentd.

* It may be better place to put SBOM in https://github.com/fluent/fluentd-docker-image
  * SBOM just reflect snapshot of container components, not fluentd itself. so, put it under fluentd-docker-image may be better IMHO.

By the way,  how about using result of tags?

```bash
`curl --silent https://hub.docker.com/v2/namespaces/fluent/repositories/fluentd/tags  | jq --raw-output '.results[] | select (.name | startswith("#{version}-debian-amd64-1.")) | .name'`
v1.16.3-debian-amd64-1.0
```

Reply:
Ah, it's really good. And I agree to put SBOMs in fluent/fluent-docker-image instead of here.
Or, like as pixie, we need to implement SBOM like auto generated files in the future.
https://github.com/pixie-io/pixie/tree/main/tools/licenses

Reply:
https://github.com/fluent/fluentd-docker-image/pull/370